CHANGELOG.md

```diff
@@ -35,7 +36,7 @@ Republished without changes to fix missing README.md in crates.io.
 - Fix rustdoc warnings
 - Upgrade nrf51-hal to 0.12.1
 
-[Unreleased]: https://github.com/nrf-rs/microbit/compare/v0.10.1...HEAD
+[unreleased]: https://github.com/nrf-rs/microbit/compare/v0.10.1...HEAD
```

Comment:
That seems wrong, the link is really using Unreleased as anchor.

Reply:
Autoformatting...arrgh